monitor/tests/test_tasks.py

```
@@ -0,0 +1,224 @@
+        assert "TestAPI" in summary
+        assert "80.0%" in summary  # Success rate
+        assert "1" in summary  # Anomaly count
+        assert "Recommendations" in summary
+    
+    def test_generate_summary_perfect_service(self):
+        """Test summary for perfectly healthy service."""
+        service = baker.make(Service, name="PerfectAPI")
+        
+        # All successful results
+        results = []
+        for i in range(10):
```

Ruff (0.12.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)